README.md

```diff
@@ -18,314 +18,245 @@ Visit the full docs [here](https://PrefectHQ.github.io/prefect-dask) to see addi
         <img src="https://img.shields.io/badge/discourse-browse_forum-red.svg?color=0052FF&labelColor=090422&logo=discourse" /></a>
 </p>
 
-## Welcome!
-
-Prefect integrations with the [Dask.distributed](http://distributed.dask.org/) library for distributed computing in Python.
-
-Provides a `DaskTaskRunner` that enables flows to run tasks requiring parallel or distributed execution using Dask.
-
-## Getting Started
-
-### Python setup
-
-Requires an installation of Python 3.7+.
-
-We recommend using a Python virtual environment manager such as pipenv, conda, or virtualenv.
-
-These tasks are designed to work with Prefect 2.0. For more information about how to use Prefect, please refer to the [Prefect documentation](https://orion-docs.prefect.io/).
-
-### Installation
-
-Install `prefect-dask` with `pip`:
-
-```bash
-pip install -U prefect-dask
-```
-
-## Running tasks on Dask
-
-The `DaskTaskRunner` is a parallel task runner that submits tasks to the [`dask.distributed`](http://distributed.dask.org/) scheduler. 
-
-By default, a temporary Dask cluster is created for the duration of the flow run.
-
-For example, this flow counts up to 10 in parallel (note that the output is not sequential).
-
-```python
-import time
-
-from prefect import flow, task
-from prefect_dask import DaskTaskRunner
-
-@task
-def shout(number):
-    time.sleep(0.5)
-    print(f"#{number}")
-
-@flow(task_runner=DaskTaskRunner)
-def count_to(highest_number):
-    for number in range(highest_number):
-        shout.submit(number)
-
-if __name__ == "__main__":
-    count_to(10)
-
-# outputs
-#3
-#7
-#2
-#6
-#4
-#0
-#1
-#5
-#8
-#9
-```
-
-If you already have a Dask cluster running, either local or cloud hosted, you can provide the connection URL via an `address` argument.
+Visit the full docs [here](https://PrefectHQ.github.io/prefect-dask) to see additional examples and the API reference.
 
-To configure your flow to use the `DaskTaskRunner`:
+The `prefect-dask` collection makes it easy to include distributed processing for your flows. Check out the examples below to get started!
 
-1. Make sure the `prefect-dask` collection is installed as described earlier: `pip install prefect-dask`.
-2. In your flow code, import `DaskTaskRunner` from `prefect_dask.task_runners`.
-3. Assign it as the task runner when the flow is defined using the `task_runner=DaskTaskRunner` argument.
+## Integrate with Prefect flows
 
-For example, this flow uses the `DaskTaskRunner` configured to access an existing Dask cluster at `http://my-dask-cluster`.
+Perhaps you're already working with Prefect flows. Say your flow downloads many images to train your machine learning model. Unfortunately, the it takes a long time to download your flows because your code is running sequentially.
```

Comment:
```suggestion
Perhaps you're already working with Prefect flows. Say your flow downloads many images to train your machine learning model. Unfortunately, it takes a long time to download your flows because your code is running sequentially.
```